source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ADCChoiceSetResourceResolver.m

```diff
@@ -0,0 +1,30 @@
+//
+//  ADCChoiceSetResourceResolver.m
+//  ADCIOSVisualizer
+//
+//  Created by Ankit Maurya on 9/21/18.
+//  Copyright © 2018 Microsoft. All rights reserved.
+//
+
+#import "ADCChoiceSetResourceResolver.h"
+
+@implementation ADCChoiceSetResourceResolver
+
+- (UIImage*) getRadioButtonForSelectedState{
+    return [UIImage imageNamed:@"checked.png" inBundle:[NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"] compatibleWithTraitCollection:nil];
```

Comment:
is it possible to extract these into instance variables like radioButtonCheckedImage, radioButtonUncheckedImage, etc and initialize only once and return everytime?